about.html

```diff
@@ -23,8 +23,20 @@
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
-    <!-- Menu Mobile-->
-    <navbar class="nav"><img src="images/mobile_menu.png" alt="hamburger"></navbar>
+    <nav>
+      <a class="hamburger">
+        <img src="images/mobile_menu.png" alt="hamburger">
+      </a>
+    </nav>
+    <nav>
+
+      <ul class="mobile-menu">
+        <a class="close"><img src="images/cancel.png" alt=""></a>
+        <li class="menu-link"><a href="index.html">Home</a></li>
+        <li class="menu-link"><a href="about.html">About</a></li>
+        <li class="menu-link menu-highlight"><a href="#">Stoa</a></li>
+      </ul>
+    </nav>
   </header>
 
 
```

Comment:
- [ ] Kindly consider adding a divider after the Headline of the middle section
- [ ] Kindly consider adding a horizontal line after the middle section so it looks identical to the provided design

|Your page|Provided design|
|-|-|
|![Screenshot_20230318_001502](https://user-images.githubusercontent.com/25227022/226067954-835fb0b1-75df-4c28-a7b4-1c806898c370.png)|![Screenshot_20230318_001741](https://user-images.githubusercontent.com/25227022/226068064-87e6733f-329c-4fe4-8e21-b1d4e36eb66a.png)|